.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Ruby Gem
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Ruby 2.6
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.4.x
```

Comment:
Ruby 2.6 vs 2.4.x? Does that matter?

Reply:
it just sets the version to run on, since we run on 2.3 and 2.4, i set it to that.